### DISCUSSION TITLE
Unsupported Model, Select A Depth Model

### DISCUSSION THREAD

**Discussion by DataJuggler** [carson-katri/dream-textures#491] [Q&A]
This model was selected for me after I installed the Recommended Stable Diffusion 2.1. 

What do I have to do to fix this?

![image](https://user-images.githubusercontent.com/26331133/210139009-1aafcc6b-2dea-4666-8aa0-0793a9960074.png)

Thank you

**Comment by carson-katri** [comment]
Follow those instructions it gives you. Search for that depth model, then download and select it in the drop-down.

**Reply by DataJuggler** [reply]
Thank you.

**Reply by iSenChi** [reply]
Sadly doesn't work.  Any ideas about what's wrong ? Thanks.

![image](https://user-images.githubusercontent.com/15358434/212534474-2c22ebee-f355-4218-8820-b6bb5208730e.png)

**Reply by carson-katri** [reply]
@iSenChi Then the model download must have been corrupted or failed somehow. You should delete and redownload it.

**Comment by DataJuggler** [comment]
Where do I search for a Depth Model?

**Reply by carson-katri** [reply]
In the addon preferences

**Reply by Esimacio** [reply]
is there any clear answer to this yet?

**Reply by carson-katri** [reply]
@Esimacio The instructions for model setup are here: https://github.com/carson-katri/dream-textures/blob/main/docs/SETUP.md#download-a-model

**Reply by dearwood777** [reply]
still not worked~
